packages/react-components/review/beta/react-components.api.md

```diff
@@ -467,7 +474,7 @@ export interface ChatMessage extends MessageCommon {
     // (undocumented)
     failureReason?: string;
     // @beta
-    files?: FileMetadata[];
+    files?: AttachmentMetadata[];
```

Comment:
we should rename files to attachments to prevent further confusions on terminologies

Reply:
> we should rename files to attachments to prevent further confusions on terminologies

I'll have a separate PR for it afterwards. Thanks for pointing it out.